lib/autoupdate/agent/config.go

```diff
@@ -59,8 +60,8 @@ type UpdateSpec struct {
 	Proxy string `yaml:"proxy"`
 	// Group specifies the update group identifier for the agent.
 	Group string `yaml:"group,omitempty"`
-	// URLTemplate for the Teleport tgz download URL.
-	URLTemplate string `yaml:"url_template,omitempty"`
+	// BaseURL is CDN base URL used in `url_template` for the Teleport tgz download URL.
+	BaseURL string `yaml:"base_url,omitempty"`
```

Comment:
This looks like a breaking change? Was the template feature backported already?

Reply:
only for client tools, they have separate PRs, where agent logic is not affected
v17: https://github.com/gravitational/teleport/pull/51476
v16: https://github.com/gravitational/teleport/pull/51478
v15: https://github.com/gravitational/teleport/pull/51482